test/transform-tests.js

```diff
@@ -15,6 +19,8 @@ describe("compiler.transform", () => {
       const code = files[absPath];
       const ast = options.parse(code);
 
+      options.moduleId = makeUniqueId("module", code);
```

Comment:
Bikeshed, `moduleId` is kind of a odd name. When I think of module id I think of the identifier of a module. Like `reify` is the module id of `require('reify')`. Maybe `moduleAlias`?